tests/cases/compiler/es5-asyncFunctionTryStatements.ts

```diff
@@ -1,10 +1,10 @@
 // @lib: es5,es2015.promise
 // @noEmitHelpers: true
 // @target: ES5
-declare var x, y, z, a, b, c;
+declare var x: any, y: any, z: any, a: any, b: any, c: any;

```

Comment:
I thought that auto typing didn't apply to ambient contexts. Why did you have to add `any` here?